lib/web/ui/integration.go

```diff
@@ -234,6 +234,29 @@ type AWSOIDCDeployDatabaseServiceResponse struct {
 	ClusterDashboardURL string `json:"clusterDashboardUrl"`
 }
 
+// AWSOIDCEnrollEKSClustersRequest is a request to ListEKSClusters using the AWS OIDC Integration.
+type AWSOIDCEnrollEKSClustersRequest struct {
+	Region             string   `json:"region"`
+	ClusterNames       []string `json:"clusterNames"`
+	JoinToken          string   `json:"joinToken"`
+	ResourceID         string   `json:"resourceId"`
+	EnableAppDiscovery bool     `json:"enableAppDiscovery"`
```

Comment:
I wonder if we should allow for different configurations :thinking: 
Let's say I have 3 clusters but I only want to cofigure App Discovery in one of them
Or I want to use different JoinToken per cluster

Btw, is JoinToken configurable in the frontend?

Reply:
In that case I would set the `ClusterName` variable to be a string instead of a list of strings.

Up to you :+1: